src/main/java/carracing/domain/CarFactory.java

```diff
@@ -0,0 +1,19 @@
+package carracing.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarFactory {
+
+    private CarFactory() {
+    }
+
+    public static List<Car> makeCars(int count) {
+        List<Car> cars = new ArrayList<>(count);
+        for (int i=0; i<count; i++){
```

Comment:
```suggestion
        for (int i = 0; i < count; i++) {
```

컨벤션을 위반하셨습니다. IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요.

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.

Reply:
매번 한다는게 까먹고 실수하네요 ㅜㅜ
지금부터라도 습관화 해야겠습니다 :)